src/TestFramework/TestFramework/Interfaces/ITestDataSourceUnfoldingCapability.cs

```diff
@@ -22,17 +22,34 @@ public enum TestDataSourceUnfoldingStrategy : byte
     /// <summary>
     /// MSTest will decide whether to unfold the parameterized test based on value from the assembly level attribute
     /// <see cref="TestDataSourceOptionsAttribute" />. If no assembly level attribute is specified, then the default
-    /// configuration is to unfold.
+    /// configuration is to unfold using <see cref="System.Runtime.Serialization.Json.DataContractJsonSerializer"/>.
     /// </summary>
     Auto,
 
     /// <summary>
     /// Each data row is treated as a separate test case.
     /// </summary>
+    [Obsolete("Use 'UnfoldUsingDataContractJsonSerializer' instead")]
```

Comment:
Consider EditorBrowsable(Never), and maybe also inheritdoc from `UnfoldUsingDataContractJsonSerializer`